src/Phantomjs.coffee

```diff
@@ -4,15 +4,16 @@ ChildProcess = require './ChildProcess'
 EventEmitter = require('events').EventEmitter
 path = require 'path'
 phantomjs = require 'phantomjs'
-process.env.PATH = path.dirname(phantomjs.path) + ':' + process.env.PATH
 isCoffee = __filename.indexOf('.coffee') > 0
 
+DEFAULT_PATH = process.env.PATH
```

Comment:
Don't set it here, just set process.env.PATH inside Phantomjs, so there are no test side effects